franka_ros/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Why is this not updated to 3.10 as well?

Reply:
Hey @gollth, thanks for reviewing my PR! I got an error when I updated it to 3.10 since the ROS meta package specs currently require version 2.8.3 (see https://wiki.ros.org/catkin/package.xml#Metapackages and https://github.com/ros-planning/moveit/issues/1619). 

#### How to reproduce

1. Checkout https://github.com/frankaemika/franka_ros/commit/22627eff64b2f186dc713ab214a359d78a859fcb#diff-64225d9dae606e899687341a27336f3cfff2889d702f52095a148a4e6d50cb24 or change the CMake version in the `franka_ros/CMakeLists.txt` file to `VERSION 3.10`.
2. Use `catkin_make` to build the package. 
3. Be greeted by the following error:

```bash
WARNING: The CMakeLists.txt of the metapackage 'franka_ros' contains non standard content. Use the content of the following file instead: /home/ricks/development/work/franka_ros_ws/build/catkin_generated/metapackages/franka_ros/CMakeLists.txt
``` 

> [!NOTE]\
> Catkin-tools doesn't have this problem.